.pre-commit-config.yaml

```
@@ -134,3 +134,9 @@ repos:
         exclude: >-
           ^docs/changes/(\d+\.other.rst|README.rst|template.rst)
         files: ^docs/changes/\d+.\w+.rst
+
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.2
+    hooks:
+      - id: codespell
+        args: ["--toml pyproject.toml"]
```

```suggestion
  - repo: https://github.com/codespell-project/codespell
    rev: v2.2.2
    hooks:
      - id: codespell
        args: ["--toml=pyproject.toml"]
        additional_dependencies:
          - tomli
```

I was experimenting with setting up `codespell` for one of my other project including using the pre-commit hook (inspired by this and #13982). I found that I needed to modify this so that it would actually use the `toml` settings, these were the necessary changes.

---

Thank you!  I'll merge this, and then pose a question about `--write-changes` below.